packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -1044,14 +1124,14 @@ export class GarbageCollector implements IGarbageCollector {
 
 	/**
 	 * Generates the stats of a garbage collection sweep phase run.
-	 * @param allDeletedNodes - All the nodes that have been deleted across all GC runs.
-	 * @param currentDeletedNodes - The nodes that have been deleted in this GC run.
+	 * @param deletedNodes - The nodes that have been deleted until this run.
+	 * @param sweepReadyNodes - The nodes that are sweep ready in this GC run.
 	 * @param markPhaseStats - The stats of the mark phase run.
 	 * @returns the stats of the sweep phase run.
 	 */
 	private getSweepPhaseStats(
-		allDeletedNodes: Set<string>,
-		currentDeletedNodes: Set<string>,
+		deletedNodes: Set<string>,
```

Comment:
FYI you drop this param and just use `this.deletedNodes`.  It's fine this way too IMO.